### PR TITLE
[KV] Metrics & Analytics docs for KV

### PR DESCRIPTION
**Pull request by thomasgauvin**
[DO NOT MERGE]

### CODE REVIEW

File: data/changelogs/kv.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+productAreaLink: /workers/platform/changelog/platform/
+entries:
+  - publish_date: "2024-08-08"
+    title: New KV Analytics API
+    description: |-
+      Workers KV now has a new [analytics API](/kv/observability/metrics-analytics/) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics on both operations and storage, including latency metrics for both read and write operations to Workers KV.
+
+      The metrics queried by this API will power upcoming improvements to the Workers KV dashboard analytics.
+
+      The legacy Workers KV [analytics REST API](/api/operations/workers-kv-request-analytics-query-request-analytics) will be turned off as of January 31st, 2025. Developers using this API will receive a series of email notifications prior to the shutdown of the legacy API.
```
Comment (thomasgauvin) [inline data/changelogs/kv.yaml:15]:
```suggestion
  - publish_date: "2024-08-08"
    title: New KV Analytics API
    description: |-
      Workers KV now has a new [metrics dashboard](/kv/observability/metrics-analytics/#view-metrics-in-the-dashboard) and [analytics API](/kv/observability/metrics-analytics/#query-via-the-graphql-api) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics for both operations and storage, including latency metrics for read and write operations to Workers KV.

      The legacy Workers KV [analytics REST API](/api/operations/workers-kv-request-analytics-query-request-analytics) will be turned off as of January 31st, 2025. Developers using this API will receive a series of email notifications prior to the shutdown of the legacy API.
```

@elithrar was thinking of only merging when dashboard is complete to minimize unnecessary back & forth given this is probably 1-2 weeks away. Open to either, will need to remove section #view-metrics-in-the-dashboard if we decide to incrementally make this change.

File: content/kv/observability/metrics-analytics.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    "accountTag":"<YOUR_ACCOUNT_ID>",
+    "namespaceId": "<YOUR_KV_NAMESPACE_ID>",
+    "date_geq": "2024-07-15",
+    "date_leq": "2024-07-30"
+}
+```
+
+#### Operations
+
+To query the sum of read, write, delete, and list operations for a given `namespaceId` and for a given date range (`date_geq` and `date_leq`), grouped by `date` and `action:
```
Comment (kodster28) [inline content/kv/observability/metrics-analytics.md]:
```suggestion
To query the sum of read, write, delete, and list operations for a given `namespaceId` and for a given date range (`date_geq` and `date_leq`), grouped by `date` and `action`:
```

**Review by kodster28 — APPROVED** [review]
Changelog looks good + looked through the metrics / analytics page and it seems clear to me.

**Comment by kodster28** [comment]
Let's go ahead and close this out since we've got the replacement one up

**Comment by thomasgauvin** [comment]
Closed in favor of https://github.com/cloudflare/cloudflare-docs/pull/16328

**Pull request closed without merging**